synapse/python_dependencies.py

```diff
@@ -75,6 +75,7 @@
     "Jinja2>=2.9",
     "bleach>=1.4.3",
     "typing-extensions>=3.7.4",
+    "setuptools-scm",
```

Comment:
I still don't really understand why this is required. Synapse doesn't use `setuptools-scm`.

Reply:
(it might solve your problem, but that doesn't necessarily mean that it's a sensible way to solve your problem)

Reply:
@woutervb Could you provide the error output you received when `setuptools-scm` is not included?